### PR TITLE
Add no-std, no-alloc crates.io category metadata

### PR DESCRIPTION
**Pull request by lopopolo**
Prepare for v1.0.1 release.

@x-hgg-x are you ok with me publishing a v1.0.1?

### CODE REVIEW

**Comment by lopopolo** [comment]
oh whoops, I misclicked merge.

@x-hgg-x I'll wait for your signoff before releasing.

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ documentation = "https://docs.rs/strftime-ruby"
 homepage = "https://github.com/artichoke/strftime-ruby"
 description = "Ruby `Time#strftime` parser and formatter"
 keywords = ["ruby", "strftime", "time"]
-categories = ["date-and-time", "no-std", "parser-implementations", "value-formatting"]
+categories = ["date-and-time", "no-std", "no-std::no-alloc", "parser-implementations", "value-formatting"]
```
Comment (x-hgg-x) [inline Cargo.toml:15]: > `"no-std::no-alloc"`

We already have `no-std`, so `no-alloc` should be sufficient.

**Comment by x-hgg-x** [comment]
A patch release is ok in this case in order to update the crate metadata.

**Comment by lopopolo** [comment]
Tagged: https://github.com/artichoke/strftime-ruby/releases/tag/v1.0.1
Released: https://crates.io/crates/strftime-ruby/1.0.1

**Pull request merged**